samplekotlin/src/main/AndroidManifest.xml

```diff
@@ -20,9 +20,13 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".ChannelActivity" />
-        <activity android:name=".ChannelActivity2" />
-        <activity android:name=".ChannelActivity3" />
-        <activity android:name=".ChannelActivity4" />
+        <activity android:name=".ChannelActivity"
```

Comment:
Lets just change the base application theme to material:
`<style name="AppTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">`